### ISSUE TITLE
Writing options using QueryOptionsManager on uber port throws REST-UNSUPPORTEDPARAM

### ISSUE THREAD

**Issue by georgeajit** [marklogic/java-client-api#760]
Here is a sample code that causes this exception. Create a DatabaseClient instanace on port 8000.

```
public void setQueryOption(DatabaseClient client, String queryOptionName) throws FileNotFoundException
	{
		// create a manager for writing query options
		QueryOptionsManager optionsMgr = client.newServerConfigManager().newQueryOptionsManager();

		// create handle
		ReaderHandle handle = new ReaderHandle();
		
		// write the files
		BufferedReader docStream = new BufferedReader(new FileReader("src/test/java/com/marklogic/client/functionaltest/queryoptions/" + queryOptionName));
		handle.set(docStream);
			
		//handle.setFormat(Format.XML);
		
		// write the query options to the database
		optionsMgr.writeOptions(queryOptionName, handle);		    

		System.out.println("Write " + queryOptionName + " to database");
	}
```
Stack Trace:
```
com.marklogic.client.FailedRequestException: Local message: /config/query write failed: Bad Request. Server Message: REST-UNSUPPORTEDPARAM: (rest:UNSUPPORTEDPARAM) Endpoint does not support query parameter: database
	at com.marklogic.client.impl.OkHttpServices.putPostValueImpl(OkHttpServices.java:2801)
	at com.marklogic.client.impl.OkHttpServices.putValue(OkHttpServices.java:2658)
	at com.marklogic.client.impl.QueryOptionsManagerImpl.writeOptions(QueryOptionsManagerImpl.java:158)
	at com.marklogic.client.functionaltest.BasicJavaClientREST.setQueryOption(BasicJavaClientREST.java:1233)
	at com.marklogic.client.functionaltest.TestDatabaseClientConnection.testQueryManagerTuples(TestDatabaseClientConnection.java:387)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
```

**Comment by sammefford** [comment]
This should be resolved by the same fix for #761 .  Let me know if it's not.

**Comment by georgeajit** [comment]
This issue is resolved. Tested with latest pull of 3.0 develop and on MarkLogic Server 8.0 build 06302017.
